consumer/main.go

```diff
@@ -29,6 +29,10 @@ func init() {
 	}
 }
 
+func notUsed() {
+	panic("Some sort of panic")
+}
```

Comment:
**Suggestion:** The function 'notUsed' is not used anywhere in the code and it triggers a panic which can cause the program to crash. It's better to remove this function if it's not used, or handle the panic properly if it's necessary.
```suggestion

```